contrib/manual/mantemplate

```diff
@@ -265,9 +281,261 @@ $ \fBnordvpn set dns off\fR
 $ \fBnordvpn set dns 0.0.0.0 1.2.3.4\fR
 .fi
 .RE
+
+.SH "MESHNET"
```

Comment:
1. I know that previously our manpage only had "standard" sections (like "usage", "examples"), but I couldn't think of a better way to categorize so much information about Meshnet, so I did it the way that I did. There doesn't seem to be anything like "subsections" in manpage. Anyways, any specific suggestions about the formatting are welcome.
2. ~95% is copied from Meshnet docs. The remaining 5% are some small edits by me because I cut some stuff to avoid repeatability.
3. Didn't ask for writer review on this because this is already written by a writer. Well except for my small edits, but they are very hard to track and I don't think this is worth the scrutiny :smile: But if something looks wrong do tell.